### PR TITLE
Update Colorize Theme to use TweenService and task.wait

### PR DESCRIPTION
**Pull request by xvarmkorv2**

### CODE REVIEW

File: MainModule/Client/UI/Colorize.rbxmx
Comment (P3tray) [inline MainModule/Client/UI/Colorize.rbxmx:61]: Please don't uncaptialize `:Connect()`

File: MainModule/Client/UI/Colorize.rbxmx
Comment (P3tray) [inline MainModule/Client/UI/Colorize.rbxmx:66]: Please don't uncaptialize `:Connect()`

File: MainModule/Client/UI/Colorize.rbxmx
Comment (P3tray) [inline MainModule/Client/UI/Colorize.rbxmx:103]: We recently removed all pairs() and ipairs() functions from our code.

Pleas change `for i,v in ipairs(obj:GetChildren())` to `for i,v in obj:GetChildren()` and re-test your code.

**Review by P3tray — CHANGES_REQUESTED** [review]
If the code has been tested by yourself and is working, then please make these further changes.

**Review by LawMixer — APPROVED** [review]
I mean, this looks good to me. Just wait for somebody with write access to review and merge it.

**Review by Expertcoderz — APPROVED** [review]

**Pull request merged**
